src/containers/edit-profile/notification-tab/NotificationContainer.tsx

```diff
@@ -0,0 +1,38 @@
+import { useTranslation } from 'react-i18next'
+
+import Box from '@mui/system/Box'
+import Switch from '@mui/material/Switch'
+
+import SettingItem from '~/components/setting-item/SettingItem'
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import { titleWithSubtitle } from '~/containers/edit-profile/notification-tab/NotificationContainer.constans'
+import { styles } from '~/containers/edit-profile/notification-tab/NotificationContainer.style'
+
+const NotificationContainer = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Box sx={styles.root}>
+      <TitleWithDescription
+        description={t('editTutor.notificationTab.mainHint')}
+        style={styles.titleWithDescription}
+        title={t('editTutor.notificationTab.notifications')}
+      />
+
+      <Box sx={styles.optionsContainer}>
+        {titleWithSubtitle.map((item) => (
+          <SettingItem
+            key={item.id}
+            style={styles.options}
+            subtitle={t(item.subtitle)}
+            title={t(item.title)}
+          >
+            <Switch sx={styles.switch} />
```

Comment:
<img width="920" alt="image" src="https://github.com/ita-social-projects/SpaceToStudy-Client/assets/56305508/0a4885cf-6956-4a14-b112-08b8efc122e5">

try to fix your ts warning